### PR TITLE
made filters clickable

### PR DESCRIPTION
**Pull request by kr1s-r**

### CODE REVIEW

File: packages/client/src/components/tabs/AnalysisTab.tsx [hunk 1]
```diff
@@ -87,14 +87,16 @@ function AnalysisTab() {
...
-                className="h-4 w-4 accent-[#B94A6C]"
-                id={filter}
-                name={filter}
-                type="checkbox"
-              />
-              <span>{filter}</span>
+            <div className="flex items-center" key={filter}>
+              <label className="cursor-pointer space-x-2">
+                <input
+                  className="h-4 w-4 accent-[#B94A6C]"
```
Comment (justin-phxm) [inline packages/client/src/components/tabs/AnalysisTab.tsx:93]:
```suggestion
                  className="size-4 accent-custom-color"
```

Can we also add this custom color to the tailwind config file?

**Review by justin-phxm — CHANGES_REQUESTED** [review]
rebase off main

**Review by justin-phxm — APPROVED** [review]

**Review by justin-phxm — CHANGES_REQUESTED** [review]
image

Can you center the checkbox and the text. Also, It should be a pointer when you hover over the checkbox...

Instead of using space-x-2, use gap-2 on the parent.

**Review by alexwhelan12 — APPROVED** [review]
LGTM!

**Pull request merged**